debug-server.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+var start = require('./census/app').start;
```

Comment:
@brew extremely small issue, but, as we are implementing ES6 going forward, `const` is better here.